x-pack/plugins/enterprise_search/public/applications/analytics/components/analytics.scss

```diff
@@ -0,0 +1,4 @@
+.entSearch__collectionIntegrateLayout {
```

Comment:
nitpick: move the scss file to the root of components/analytics_collection_view folder. Its so we can co-locate the styles with the view component.